framework/Samplers/AdaptiveSobol.py

```diff
@@ -400,6 +400,7 @@ def localGenerateInput(self,model,oldInput):
         self.inputInfo['ProbabilityWeight']*=self.inputInfo['ProbabilityWeight-'+dist]
     self.inputInfo['PointProbability'] = reduce(mul,self.inputInfo['SampledVarsPb'].values())
     # reassign SampledVarsPb to fully correlated variables
+<<<<<<< HEAD
```

Comment:
leftover of merging

Reply:
yes, actively working on this one right now.